### ISSUE TITLE
Sublime 4149 generates "Error loading syntax file[...]: no such target" pop-up for "JavaScript (Babel)"

### ISSUE THREAD

**Issue by nabijaczleweli** [babel/babel-sublime#427]
![](https://user-images.githubusercontent.com/6709544/236218069-2667b80d-104b-42a3-a014-26d7499db321.png)

And, indeed, I see a 
```yaml
            -   - meta_include_prototype: false
                - match: ''
                  set: scope:source.js.css#at-keyframe-block-content
                  with_prototype:
                    - match: (?=`)
                      pop: true
                    - match: \$\{
                      scope: punctuation.section.interpolation.begin.js
                      push:
                        - clear_scopes: 1  
```
block, and this is the only mention of this in the package.

**Comment by Thom1729** [comment]
I'm on vacation, but I should be able to look into it later today. My initial guess is that there's a mismatch between the version of the Styled Components syntax in JS Custom and babel-sublime. If so, the proper fix would be to update that syntax. (It's possible that the script I use to update babel-sublime from JS Custom doesn't catch that file, in which case I should fix that as well.)

It's also possible that recent changes to the core CSS syntax broke the Styled Components extension. I have not looked into this at all but I vaguely recall hearing about changes to that syntax.

It could also be something completely different.

**Comment by Thom1729** [comment]
Since it seems to be broken upstream in JS Custom as well, it's probably a breaking change to the core CSS syntax.

**Comment by Thom1729** [comment]
Fixed in v11.2.0.
